doc/source/dev/reviewer_guidelines.rst

```diff
@@ -91,6 +91,11 @@ For maintainers
   explain why. If it's not, then it's a good idea to first explain why you
   think the PR is not suitable for inclusion in NumPy and then let a second
   committer comment or close.
+- If the PR submitter doesn't respond to your comments for 6 months, move the PR 
+  in question to the inactive category with the “inactive” tag attached. At this point,
+  by a quorum of at least 2 maintainers, the PR can be closed or finalized by another 
```

Comment:
```suggestion
  by a quorum of at least 2 maintainers, the PR can be closed. If there is any interest in finalizing said PR, then this can be indicated at any time (without waiting 6 months) by a comment.   
```

There was some discussion on the [25-05-2022](https://hackmd.io/76o-IxCjQX2mOXO_wwkcpg?view) community call. As part of this, the rewording above might be more friendly.